rwengine/src/ai/CharacterController.cpp

```diff
@@ -239,6 +239,28 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
         } else if (character->animator->getAnimation(AnimIndexAction) ==
                    anm_enter) {
             if (character->animator->isCompleted(AnimIndexAction)) {
+                if (vehicle->getVehicle()->vehiclename_ == "TAXI"
+                    || vehicle->getVehicle()->vehiclename_ == "CABBIE"
+                    || vehicle->getVehicle()->vehiclename_ == "BORGNINE") {
+                    // Earn $25 from taxi cab
+                    // @todo
+                } else if (vehicle->getVehicle()->vehiclename_ == "POLICAR") {
+                    // Police cruisers give 5 shotgun cartridges
+                    character->addToInventory(4, 5);
```

Comment:
That hasn't crossed my mind, but now I'm thinking that it probably does give bonuses to non-player character objects, since it's within the `src/ai/` directory it's likely covering both player and NPCs.
What'd be the best way to make sure the character object is the player?